python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -39,6 +39,8 @@ class ForcedPhotCoaddConfig(ForcedPhotImageConfig):
         optional=True
     )
 
+    fakesAdded = lsst.pex.config.Field(dtype=bool, default=False, doc="Have fakes been added?")
```

Comment:
This Field should follow the structure of the other fields already in the file, aka multi-line

Documentation should be statements and not questions, perhapse something like "Fields indicates if Fake objects are present in input dataset"

Consider a different name for the field variable, like fakesPresent of fakesInserted as it is closer to the use of this variable in English terms, but this is just a suggestion from someone reading it, there is nothing wrong with how it is

Reply:
I've changed the structure of the Field but have let the name. I have used fakesAdded in my examples for the DESC meeting and would prefer to keep things consistent.